File: Praktikum 7/Python/PRAK605-2310817210004-Allano Lintang Ertantora.py
barisKolom = int(input())
print('Matriks A')
matrixA = []
for i in range(barisKolom):
    matrixA.append(list(map(int, input().split())))
print('Matriks B')
matrixB = []
for i in range(barisKolom):
    matrixB.append(list(map(int, input().split())))
matrixHasil = []
for i in range(barisKolom):
    matrixHasil.append([0] * barisKolom)
for i in range(barisKolom):
    for j in range(barisKolom):
        for k in range(barisKolom):
            matrixHasil[i][j] += matrixA[i][k] * matrixB[k][j]
print("Matriks AXB")
for i in matrixHasil:
    print(i)

File: Praktikum 7/C/PRAK605-2310817210004-Allano-Lintang-Ertantora.c
#include <stdio.h>
int main(){
    int barisKolom;
    scanf("%d", &barisKolom);
    printf("Matriks A:\n");
    int matrixA[barisKolom][barisKolom];
    for (int i = 0; i < barisKolom; i++){
        for (int j = 0; j < barisKolom; j++){
            scanf("%d", &matrixA[i][j]);
        }
    }
    printf("Matriks B:\n");
    int matrixB[barisKolom][barisKolom];
    for (int i = 0; i < barisKolom; i++){
        for (int j = 0; j < barisKolom; j++){
            scanf("%d", &matrixB[i][j]);
        }
    }
    int matrixHasil[barisKolom][barisKolom];
    int sum = 0;
    for (int i = 0; i < barisKolom; i++){
        for (int j = 0; j < barisKolom; j++){
            for (int k = 0; k < barisKolom; k++){
                sum += matrixA[i][k] * matrixB[k][j];
            }
            matrixHasil[i][j] = sum;
            sum = 0;
        }
    }
    printf("Matriks AXB\n");
    for(int i = 0; i < barisKolom; i++){
        for(int j = 0; j < barisKolom; j++){
            printf("%d ", matrixHasil[i][j]);
        }printf("\n");
    }
    return 0;
}